client/login/wp-login/index.jsx

```diff
@@ -636,6 +636,14 @@ export class Login extends Component {
 			document.location.search?.includes( 'wpcloud' )
 		) {
 			brandLogo = <WPCloudLogo className="login__wpcloud-logo" size={ 120 } />;
+		} else {
+			brandLogo = (
+				<WordPressLogo
+					size={ 21 }
+					className="step-container-v2__top-bar-wordpress-logo"
```

Comment:
@chriskmnds,

Instead of passing an external logo that uses a (private) StepContainerV2 class name, what if we modified the `logo` prop so it accept either a React element or `'compact'`?

Either way, we need to think of a way not to use the internal class name here.

Reply:
Not sure I follow, but I recall having borrowed the same component/class from elsewhere. 

It's also a bit of intermediate code. Most probably class will be updated as we progress with refactors. Please maybe don't change anything around right now.

Reply:
You borrowed the class from the TopBar component, and that element is used within it. The problem with this approach is that we're moving StepContainerV2 to CSS modules soon, so we can't rely on class names (and this is on purpose to prevent CSS overrides.)

Reply:
Yeah, makes sense. This has to go one way or another. When is the migration scheduled for?